searchcore/src/vespa/searchcore/proton/matching/matcher.cpp

```diff
@@ -336,8 +338,9 @@ Matcher::match(const SearchRequest &request, vespalib::ThreadBundle &threadBundl
                 adjustedDuration = 0;
             }
             _stats.updatesoftDoomFactor(request.getTimeout(), softLimit, adjustedDuration);
-            LOG(info, "Triggered softtimeout factor adjustment. request=%1.3f, doomOvertime=%1.3f, limit=%1.3f and duration=%1.3f, rankprofile=%s"
+            LOG(info, "Triggered softtimeout factor adjustment. Coverage = %lu of %u . request=%1.3f, doomOvertime=%1.3f, limit=%1.3f and duration=%1.3f, rankprofile=%s"
```

Comment:
What kind of coverage is this? Should we be more precise with the terminology in the log message?